mlflow/server/handlers.py

```diff
@@ -308,6 +308,8 @@ def _search_runs():
     run_entities = _get_store().search_runs(experiment_ids, filter_string, run_view_type,
                                             max_results, order_by, page_token)
     response_message.runs.extend([r.to_proto() for r in run_entities])
+    if run_entities.token:
```

Comment:
This was a bug where the rest store was not propagating token back to the client. The current test in `rest_store` is insufficient because it is not backed by a real store -- but this is an item on our backlog to have better end-to-end REST-y integration tests, so I left the follow-up for later. I manually tested that `search_runs` works against the new server.